### PR TITLE
Dev 168 stat component

### PR DESCRIPTION
**Pull request by Max-vS**
2 different versions of a stat component. Have displayed them in the e-lab page.

image

### CODE REVIEW

File: components/01_atoms/Stat2.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default function Stat2({ description, value, color = "white" }) {
+  const textColorClass = `text-${color}`;
```
Comment (simon-hng) [inline components/01_atoms/Stat2.tsx]: That probably doesn't work

https://tailwindcss.com/docs/content-configuration#dynamic-class-names

File: components/01_atoms/Stat.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export default function Stat({ description, value }) {
```
Comment (simon-hng) [inline components/01_atoms/Stat.tsx]: Why do we need two stat components?

Comment (simon-hng) [reply components/01_atoms/Stat.tsx]: We don't use the 01_atoms and molecule folders anymore. just `/components` is fine

File: components/01_atoms/Stat2.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+  return (
+    <div className="flex h-28 w-48 max-w-sm flex-col items-center justify-center space-y-1 rounded-xl">
```
Comment (simon-hng) [inline components/01_atoms/Stat2.tsx]: Instead you could just pass `className` as prop and do that
```suggestion
    <div className={cx("flex h-28 w-48 max-w-sm flex-col items-center justify-center space-y-1 rounded-xl", className)}>
```

**Review by simon-hng — APPROVED** [review]

File: components/Stat.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import cx from 'classnames';
```
Comment (simon-hng) [inline components/Stat.tsx]: should be an import from `cva`

**Review by simon-hng — CHANGES_REQUESTED** [review]
Can you move elab.tsx file into e-lab.tsx since thats where the hero already exists

**Review by simon-hng — APPROVED** [review]

**Review by MunzerDw — APPROVED** [review]
Top, thanks!

**Pull request merged**